metadata.yaml

```diff
@@ -0,0 +1,35 @@
+# Copyright 2022 Canonical Ltd.
+# See LICENSE file for licensing details.
+
+# For a complete list of supported options, see:
+# https://juju.is/docs/sdk/metadata-reference
+
+name: temporal-admin-k8s
```

Comment:
Should there be the "-operator" suffix?

Reply:
No, according to naming conventions: https://juju.is/docs/sdk/naming#heading--naming-charms
I guess sthe "-operator" suffix is used mostly for repo names, but not charm names.